### PR TITLE
[AKS] add UltraSSD support

### PR DESCRIPTION
**Pull request by andyzhangx**
**Description**

aks: add 'enable-ultra-ssd' parameter

```console
# docker run -u $(id -u):$(id -g) -v ${HOME}:/home/az -e HOME=/home/az --rm -it andyzhangx/azure-cli:v2.25.0-1
# az aks create -h | grep ultra
    --enable-ultra-ssd                        : Enable UltraSSD on agent node pool.
        az aks create -g MyResourceGroup -n MyManagedCluster --enable-ultra-ssd
# az aks nodepool add -h | grep ultra
    --enable-ultra-ssd             : Enable UltraSSD on agent node pool.

az aks create -g $RESOURCE_GROUP_NAME -n $CLUSTER_NAME --node-count 2 --generate-ssh-keys --kubernetes-version 1.20.5 --node-vm-size Standard_D2s_v3 --zones 1 2 3 --enable-ultra-ssd

az aks nodepool add -g $RESOURCE_GROUP_NAME --cluster-name $CLUSTER_NAME --node-count 2 --node-vm-size Standard_D2s_v3 -n ultra2 --zones 1 2 3 --enable-ultra-ssd
```

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -4484,3 +4484,22 @@ def test_aks_update_to_msi_cluster_with_addons(self, resource_group, resource_gr
...
+        })
+
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --node-vm-size Standard_D2s_v3 --zones 1 2 3 --enable-ultra-ssd'
+        self.cmd(create_cmd, checks=[
+            self.check('provisioningState', 'Succeeded')
+        ])
+
+        # delete
+        self.cmd(
+            'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:4505]: Better to leave a blank line at the end of a .py file 😄

**Review by FumingZhang — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 2]
```diff
@@ -821,11 +826,16 @@
   - name: --enable-encryption-at-host
     type: bool
     short-summary: Enable EncryptionAtHost, default value is false.
+  - name: --enable-ultra-ssd
+    type: bool
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py:830]: recommend to use get_three_state_flag() for boolean https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags

File: src/azure-cli/azure/cli/command_modules/acs/_params.py [hunk 3]
```diff
@@ -252,6 +252,8 @@ def load_arguments(self, _):
         c.argument('aci_subnet_name')
         c.argument('enable_encryption_at_host', options_list=[
                    '--enable-encryption-at-host'], action='store_true')
+        c.argument('enable_ultra_ssd', options_list=[
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/acs/_params.py:255]: so it cannot be updated to false anymore, right?

Comment (andyzhangx) [reply src/azure-cli/azure/cli/command_modules/acs/_params.py:255]: `false` is by default, so it's meaningless to set as `false`, while it's still possible

File: src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py (hunk 1 above)
Comment (andyzhangx) [reply src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py]: fixed

**Comment by FumingZhang** [comment]
@andyzhangx Better to change the title of this PR to start with "[AKS]" instead of "{AKS}" according to this [doc](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#format-pr-title), since this change is customer-facing and the message should be put into HISTORY.rst.

**Comment by andyzhangx** [comment]
> @andyzhangx Better to change the title of this PR to start with "[AKS]" instead of "{AKS}" according to this [doc](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#format-pr-title), since this change is customer-facing and the message should be put into HISTORY.rst.

done.

File: src/azure-cli/azure/cli/command_modules/acs/_params.py (hunk 3 above)
Comment (andyzhangx) [reply src/azure-cli/azure/cli/command_modules/acs/_params.py:255]: @Juliehzl could you approve since there is no blocking issue? thanks.

**Review by Juliehzl — APPROVED** [review]

**Pull request merged**